tests/e2e/e2e_suite_test.go

```diff
@@ -42,7 +40,6 @@ func init() {
 	flag.StringVar(&oc_cli, "oc_cli", "oc", "OC CLI Client")
 	flag.StringVar(&stream, "stream", "up", "[up, down] upstream or downstream")
 	flag.Int64Var(&timeoutMultiplierInput, "timeout_multiplier", 1, "Customize timeout multiplier from default (1)")
-	flag.BoolVar(&virtTestingEnabled, "test_virt", false, "Enable Virtual Machine backup/restore testing (default false)")
```

Comment:
So this flag doesn't apply anymore?

Reply:
I see.. it's in the It label
```go
	It("should upload a data volume successfully", Label("virt"), func() {
```

Reply:
Right, I am trying to reduce this part to ginkgo labels, so all the virtualization testing is triggered by the "virt" test filter.